src/status_im/multiaccounts/create/core.cljs

```diff
@@ -224,50 +222,65 @@
           :else {:db (assoc-in db [:intro-wizard :step]
                                (inc-step step))})))
 
+(defn get-selected-multiaccount [db]
+  (let [{:keys [selected-id multiaccounts]} (:intro-wizard db)]
+    (some #(when (= selected-id (:id %)) %) multiaccounts)))
+
+(defn prepare-default-account [{:keys [publicKey address]}]
+  {:public-key publicKey
+   :address    address
+   :color      colors/blue
+   :default?   true
+   :name       "Status account"})
+
 (fx/defn on-multiaccount-created
-  [{:keys [signing-phrase
-           status
-           db] :as cofx}
-   {:keys [pubkey address mnemonic installation-id
-           keycard-instance-uid keycard-key-uid keycard-pairing keycard-paired-on] :as result}
-   password
-   {:keys [seed-backed-up? login? new-multiaccount?] :or {login? true}}]
-  (let [normalized-address (utils.hex/normalize-hex address)
-        multiaccount            {:public-key             pubkey
-                                 :installation-id        (or installation-id (get-in db [:multiaccounts/new-installation-id]))
-                                 :address                normalized-address
-                                 :name                   (gfycat/generate-gfy pubkey)
-                                 :status                 status
-                                 :signed-up?             true
-                                 :desktop-notifications? false
-                                 :photo-path             (identicon/identicon pubkey)
-                                 :signing-phrase         signing-phrase
-                                 :seed-backed-up?        seed-backed-up?
-                                 :mnemonic               mnemonic
-                                 :keycard-instance-uid   keycard-instance-uid
-                                 :keycard-key-uid        keycard-key-uid
-                                 :keycard-pairing        keycard-pairing
```

Comment:
@flexsurfer this will break keycard integration. Where `keycard-key-uid`,  `keycard-pairing` and other keycard fields will be stored if they are removed from account?

Reply:
@gravityblast ?

Reply:
@flexsurfer @dmitryn I don't think we need to remove them. 
we just need to change the login with keycard so that we pass the main account address and other addresses to watch. 
why would you remove these fields @flexsurfer ?

Reply:
because i don't know where to get them from, we had them in the result from creatAccounts function before, but now we don't, that's why I mention you here

Reply:
@dmitryn @gravityblast ^

Reply:
Those fields get passed from https://github.com/status-im/status-react/blob/fd83f9ff70ca2b23dff7601fa4f9fa6f1ec530b7/src/status_im/hardwallet/core.cljs#L1611
Also, there are nullable, so if they are nil db expects it.

Reply:
oh ok, will revert then, thank you !!

Reply:
@dmitryn  done